src/main/kotlin/com/terraformation/backend/funder/FundingEntityService.kt

```diff
@@ -109,6 +109,14 @@ class FundingEntityService(
     log.info("Deleting funding entity $fundingEntityId")
 
     dslContext.transaction { _ ->
+      val funders = userStore.fetchByFundingEntityId(fundingEntityId)
+      for (funder in funders) {
+        userStore.deleteFunderById(funder.userId)
+      }
```

Comment:
This is totally valid, but my impression from discussions in Kotlin land is that if you're not doing break/continue, it's considered slightly more idiomatic to prefer `forEach`, like
```kotlin
funders.forEach { userStore.deleteFunderById(it) }
```
There's no performance difference, FYI, despite `forEach` taking a lambda function as its argument. It is declared `inline` and it compiles to exactly the same bytecode as `for/in`.

Reply:
Makes sense; I do prefer to follow idioms whenever possible. Updated!